test/e2e/e2e_test.go

```diff
@@ -56,11 +54,13 @@ func TestE2E(t *testing.T) {
 	defer logs.FlushLogs()
 	flag.Parse()
 
+	suiteConfig, _ := ginkgo.GinkgoConfiguration()
```

Comment:
Is this instantiating a separate `suitConfg` to that above here?  https://github.com/cert-manager/cert-manager/pull/5345/files#diff-d5d12b6ea07b0a78c9ad1987e1486c565ecf16839f77a93e91b09475cf6c0e99R41

We want to make sure we keep the options we set above. It looks like it is the case that it uses that same one, but can you add a comment for the next reader?

Reply:
Indeed, it's not possible to alter the config using the returned values. Instead, the updated objects have to be passed to `RunSpecs`, as explained in the docs:
```
GinkgoConfiguration returns the configuration of the current suite.
The first return value is the SuiteConfig which controls aspects of how the suite runs,
the second return value is the ReporterConfig which controls aspects of how Ginkgo's default
reporter emits output.
Mutating the returned configurations has no effect.  To reconfigure Ginkgo programmatically you need
to pass in your mutated copies into RunSpecs().
You can learn more at https://onsi.github.io/ginkgo/#overriding-ginkgos-command-line-configuration-in-the-suite
```